src/main/java/frc/robot/commands/LowerCoralElevator.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.CoralElevator;
+
+public class LowerCoralElevator extends Command {
+
+    private CoralElevator coralElevator;
```

Comment:
```suggestion
    private final CoralElevator coralElevator;
```
finalize where possible